lib/watir.rb

```diff
@@ -14,58 +14,55 @@
 require 'watir/after_hooks'
 
 module Watir
+  @relaxed_locate = true
   @always_locate = true
 
+  attr_writer :relaxed_locate, :always_locate, :default_timeout, :prefer_css, :locator_namespace
+
   class << self
-    def always_locate?
-      @always_locate
+    #
+    # Whether or not Watir should wait for an element to be found or present
+    # before taking an action.
+    # Defaults to true.
+    #
+
+    def relaxed_locate?
+      @relaxed_locate
     end
 
     #
     # Whether or not Watir should cache element references or always re-locate an Element on use.
     # Defaults to true.
     #
 
-    def always_locate=(bool)
-      @always_locate = bool
-    end
-
-    def default_timeout
-      @default_timeout ||= 30
+    def always_locate?
+      @always_locate
     end
 
     #
     # Default wait time for wait methods.
     #
 
-    def default_timeout=(value)
-      @default_timeout = value
-    end
-
-    def prefer_css?
-      @prefer_css
+    def default_timeout
+      @default_timeout ||= 30
```

Comment:
These are not implicit waits (see distinction in main conversation)

The whole point of this is to be able to remove `#when_present` scattered everywhere in a code base, which is how most people have to do it right now. 

`#default_timeout` is the default for how long you wait for an element to show up when you might need to wait for it, and I think it would only be confusing to add a new variable that from a user/non-implementation standpoint does the same thing. Either 30 seconds is too long for everything and we should adjust the default downward, or this implementation should default to the same 30 seconds.